WcaOnRails/db/seeds/development/users.seeds.rb

```diff
@@ -31,20 +31,15 @@ def self.random_user
 
 # Create members and leaders for every WCA team
 UsersController.WCA_TEAMS.each do |team|
-  leader_team = :"#{team}_leader"
-  leader = User.new(random_user.merge(
-    team => true,
-    leader_team => true,
-    name: team.to_s,
-  ))
-  leader.skip_confirmation!
-  leader.save!
+  team_id = Team.find_by_friendly_id(team).id
```

Comment:
Nitpick: This took me a couple of readthroughs to figure out, and that was because I thought the `team` variable referred to a team object from the database. How about we rename `team` to `team_friendly_id`?

It's a bummer that we have this `WCA_TEAMS` varaible that actually contains team friendly ids, not "teams".